tutorials/preprocessing/plot_40_artifact_correction_ica.py

```diff
@@ -585,7 +585,57 @@
 # .. _`qrs`: https://en.wikipedia.org/wiki/QRS_complex
 # .. _`this EEGLAB tutorial`: https://labeling.ucsd.edu/tutorial/labels
 
+
+###############################################################################
+# Example: Compute ICA components on epochs
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# ICA is now fit to MEG epochs data instead of the raw data.
+# We assume that the non-stationary EOG artifacts have already been removed.
+# The sources matching the ECG are automatically found and displayed.
+#
+# .. note::
+#     This example does quite a bit of processing, so even on a
+#     fast machine it can take about a minute to complete.
+#
+# Read and preprocess the data. Preprocessing consists of:
+#
+# - MEG channel selection
+# - 1-30 Hz band-pass filter
+# - epoching -0.2 to 0.5 seconds with respect to events
+# - rejection based on peak-to-peak amplitude
+
+filt_raw.pick_types(meg=True, eeg=False, exclude='bads', stim=True).load_data()
+filt_raw.filter(1, 30, fir_design='firwin')
+
+# peak-to-peak amplitude rejection parameters
+reject = dict(grad=4000e-13, mag=4e-12)
+# longer + more epochs for more artifact exposure
```

Comment:
```suggestion
# create longer and more epochs for more artifact exposure
```

What do you mean with "more"? Is there an example above where less epochs get selected? (I tried to look but might have missed it)